packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -571,20 +575,9 @@ function dispatchAfterDetachedBlur(target: HTMLElement): void {
   }
 }
 
-// This is a specific event for the React Flare
-// event system, so event responders can act
-// accordingly to a DOM node being unmounted that
-// previously had active document focus.
 export function beforeRemoveInstance(
   instance: Instance | TextInstance | SuspenseInstance,
 ): void {
-  if (
-    (enableDeprecatedFlareAPI || enableUseEventAPI) &&
-    selectionInformation &&
-    instance === selectionInformation.focusedElem
-  ) {
-    dispatchBeforeDetachedBlur(((instance: any): HTMLElement));
-  }
   if (enableUseEventAPI) {
```

Comment:
Is there any way we can get rid of the block below, and thus of this whole host method, now?

Reply:
Unfortunately not, this is separate logic that can't move to the other block.